google-cloud-bigquerystorage/src/test/java/com/google/cloud/bigquery/storage/v1beta2/st/STBigQueryStorageLongRunningWriteTest.java

```diff
@@ -0,0 +1,211 @@
+                Schema.of(
+                    com.google.cloud.bigquery.Field.newBuilder("foo", LegacySQLTypeName.STRING)
+                        .setMode(Field.Mode.NULLABLE)
+                        .build())))
+            .build();
+    com.google.cloud.bigquery.Field.Builder innerTypeFieldBuilder =
+        com.google.cloud.bigquery.Field.newBuilder(
+            "inner_type",
+            LegacySQLTypeName.RECORD,
+            com.google.cloud.bigquery.Field.newBuilder("value", LegacySQLTypeName.STRING)
+                .setMode(Field.Mode.REPEATED)
+                .build());
+
+    bigquery.create(tableInfo);
+    tableId =
+        String.format(
+            "projects/%s/datasets/%s/tables/%s",
+            ServiceOptions.getDefaultProjectId(), DATASET, TABLE);
+    LOG.info(
+        String.format(
+            "%s tests running with parent project: %s",
+            ITBigQueryStorageLongRunningTest.class.getSimpleName(), parentProjectId));
+  }
+
+  @AfterClass
+  public static void afterClass() {
+    if (client != null) {
+      client.close();
+    }
+    if (bigquery != null) {
+      RemoteBigQueryHelper.forceDelete(bigquery, DATASET);
+      LOG.info("Deleted test dataset: " + DATASET);
+    }
+  }
+
+  @Test
+  public void testDefaultStream() throws IOException, InterruptedException, ExecutionException,
+      Descriptors.DescriptorValidationException {
+    // Set up a default stream. Write to it for a long time, (a few minutes for now) and make
+    // sure that everything goes well.
+    String tableName = "JsonTableDefaultStream";
+    TableInfo tableInfo =
+        TableInfo.newBuilder(
+            TableId.of(DATASET, tableName),
+            StandardTableDefinition.of(
+                Schema.of(
+                    com.google.cloud.bigquery.Field.newBuilder(
+                        "test_str", StandardSQLTypeName.STRING)
+                        .build(),
+                    com.google.cloud.bigquery.Field.newBuilder(
+                        "test_numerics", StandardSQLTypeName.NUMERIC)
+                        .setMode(Field.Mode.REPEATED)
+                        .build(),
+                    com.google.cloud.bigquery.Field.newBuilder(
+                        "test_datetime", StandardSQLTypeName.DATETIME)
+                        .build())))
+            .build();
+    bigquery.create(tableInfo);
+    TableName parent = TableName.of(ServiceOptions.getDefaultProjectId(), DATASET, tableName);
+    try (JsonStreamWriter jsonStreamWriter =
+        JsonStreamWriter.newBuilder(parent.toString(), tableInfo.getDefinition().getSchema())
+            .createDefaultStream()
+            .setBatchingSettings(
+                StreamWriter.Builder.DEFAULT_BATCHING_SETTINGS
+                    .toBuilder()
+                    .setRequestByteThreshold(1024 * 1024L) // 1 Mb
+                    .setElementCountThreshold(2L)
+                    .setDelayThreshold(Duration.ofSeconds(2))
```

Comment:
Are these settings needed? I believe we want to move to the new client soon, which doesn't support these configuration afaik.

Reply:
Probably not, just removed.